docs/variants/novacustom_nv4x_adl/releases.md

```diff
@@ -13,6 +13,82 @@ For details about our release process please read
 
 </center>
 
+## v1.7.2 - 2023-12-22
```

Comment:
@mkopec we need to fix this date as well

Reply:
done